artifacts/source_type.py

```diff
@@ -426,14 +426,23 @@ def DeregisterSourceType(cls, source_type_class):
 
   @classmethod
   def GetSourceTypes(cls):
-    """Retriees the source types.
+    """Retrieves the source types.
 
     Returns:
       A list of source types (subclasses of SourceType).
     """
     return cls._source_type_classes.values()
 
   @classmethod
+  def GetSourceTypeIndicators(cls):
```

Comment:
typeindicator sounds like it's going to return an some sort of indicator object to me, but OK, stick with consistently weird.